tests/performance/codecs_float_insert.xml

```diff
@@ -10,6 +11,7 @@
                 <value>NONE</value> <!-- as a baseline -->
                 <value>LZ4</value>
                 <value>ZSTD</value>
+                <value>DEFLATE_QPL</value>
```

Comment:
Performance tests run also on ARM (search for something like "PerformanceComparisonAarch-0" in the CI status). So this won't end well 😄 

Unfortunately, I can't find a way to limit performance tests to certain platforms ... I guess you'd have to roll back your changes in the performance test .xml files for now...

Reply:
Done